doc/authentication-backends/JWT-authentication-module.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+JWT authentication backend can verify JSON Web Tokens provided by the clients.
+A wide range of signature algorithms is supported, including those using public key cryptography.
+Not only the signature is checked, but also the validity of `exp` (expired token is rejected), `iat` (token must be issued in the past) and `nbf` (token might be invalid *yet*) parameters.
```

Comment:
The module checks the signature and also the validity of the following parameters:

-  `exp` - an expired token is rejected, 
-  `iat` - a token must be issued in the past,
-  `nbf` - a token might not be valid *yet*.